src/drivers/dw/dma.c

```diff
@@ -146,8 +146,6 @@ static struct dma_chan_data *dw_dma_channel_get(struct dma *dma,
 
 		dma->chan[i].status = COMP_STATE_READY;
 
-		atomic_add(&dma->num_channels_busy, 1);
-
```

Comment:
We may remove this field. However, I think `dma_get` does have that "exclusive" flag that it can take. How is that implemented? Or is that going away too?

Reply:
Exclusive uses ref count and basically allows only one channel per DMA.

Reply:
Ah, right.